src/views/home/index.tsx

```diff
@@ -192,10 +193,12 @@ function MainView({ demos }: { demos: Demo[] }) {
 }
 
 export default function HomeViewAsyncWrapper() {
+  const [demoPath, _setDemoPath] = useStore("demoPath");
+
   return (
     <Async
       promiseFn={getDemosInDirectory}
-      args={["/home/rasmus/steamapps-common/Team Fortress 2/tf/demos"]}
+      args={[demoPath ?? "C:\\Program Files (x86)\\Steam\\steamapps\\common\\Team Fortress 2\\tf\\demos"]}
```

Comment:
Fallback to give windows users a sane default.  Could certainly be changed to a platform-specific default value

Reply:
Absolutely. Sane platform-specific defaults are a must.